src/util/system.cpp

```diff
@@ -702,23 +702,9 @@ fs::path GetDefaultDataDir()
 #endif
 }
 
-static fs::path g_blocks_path_cache_net_specific;
```

Comment:
happy to see this code simplified, it has been a frequent source of errors in the past